src/main/java/gyro/google/storage/BucketCors.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2019, Perfect Sense, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.google.storage;
+
+import com.google.api.services.storage.model.Bucket.Cors;
+import gyro.core.resource.Diffable;
+import gyro.core.resource.Updatable;
+import gyro.core.validation.ValidStrings;
+import gyro.google.Copyable;
+
+import java.util.List;
+
+/**
+ * Subresource for setting of Bucket.Cors configuration for assets within a Bucket.
+ */
+public class BucketCors extends Diffable implements Copyable<Cors> {
+
+    private Integer maxAgeSeconds;
+    private List<String> method;
+    private List<String> origin;
+    private List<String> responseHeader;
+
+    /**
+     * The value quantified in seconds to be returned in the "Access-Control-Max-Age" header.
```

Comment:
So that it renders better in the generated docs.

```suggestion
     * The value quantified in seconds to be returned in the ``Access-Control-Max-Age`` header.
```

Reply:
Done.